src/__init__.py

```diff
@@ -12,23 +12,22 @@
 
 from src.config import config
 from src.models import db
+from src.models.schema import chain, contract_oracle
 from src.network import chains
 from src.pql.exceptions import PqlDecodingError
 from src.pql.parser import parse_and_execute
+from src.utils.server import validate_request
 
 
 def create_app(args={}) -> Sanic:  # noqa: C901
     app = Sanic("src")
     app.config.from_object(config)
-    app.update_config(args)
```

Comment:
The version of `sanic` which is compatible with `gino-sanic` does not have the `update_config` method.  I recognise that this now prevents the cli arguments from propagating through to the `sanic` application.

Reply:
Let's just say 'fuck it' and go with SQLAlchemy? I had some issues with `gino-sanic` as mentioned above.

Reply:
'fuck it ' indeed.  I will purge `gino-sanic` from the existence of this repo.